fritz.defaults.yaml

```diff
@@ -436,6 +459,16 @@ skyportal:
     cosmology: Planck18
     #cosmology: {H0: "65.0", Om0: 0.3, Ode0: 0.7, name: 'skyportal_user_cosmo'}
 
+    # The minimum signal-to-noise ratio/ n-sigma for lim mag cacluations to
+    # consider a photometry point as a detection
+    photometry_detection_threshold_nsigma: 3.0
```

Comment:
Doesn't this already exist in the skyportal config?

Reply:
Yes but since its a subsection it doesn't merge.

Reply:
I don't mind adding it but it is likely going to get deleted when we finally get the gang together and figure out the whole merging config thing.